boards/redboard_artemis_nano/src/main.rs

```diff
@@ -143,6 +149,18 @@ pub unsafe fn reset_handler() {
     )
     .finalize(components::gpio_component_buf!(apollo3::gpio::GpioPin));
 
+    // Create a shared virtualisation mux layer on top of a single hardware
+    // alarm.
+    let mux_alarm = static_init!(
+        MuxAlarm<'static, apollo3::stimer::STimer>,
+        MuxAlarm::new(&apollo3::stimer::STIMER)
+    );
+    hil::time::Alarm::set_client(&apollo3::stimer::STIMER, mux_alarm);
```

Comment:
There is also `AlarmMuxComponent` for instantiating the `MuxAlarm` (sorry I know this feels kind of silly, but having components for as much initialization is the first step towards making it easy to reconfigure boards, so I am trying to enforce use of components whereever possible on new boards being added to the repo)